### ISSUE TITLE
Performance of reproduced MoCo-v2

### ISSUE THREAD

**Issue by limbo0000** [facebookresearch/vissl#240]
Thanks for your wonderful codebase!  It is more convenient to conduct various experiments. 

But I have a question about the reported MoCo-v2 in your MODEL_ZOO. It achieved 66.4% top-1 accuracy for linear classification while the official implementation of MoCo-v2 had 67.7%. Considering the std of MoCo-v2 is very small (around 0.1%), I think the gap (1.3%) between different implementation might suggest that there are some misalignments (\eg hyperparameters). Do you have any idea?

**Comment by prigoyal** [comment]
Hi @limbo0000 , thank you for reaching out about this and a very important point. I anticipate this has to do with the hyperparams we used or how we trained the model (for example #gpus, batch size etc).

**Comment by limbo0000** [comment]
Thanks for your quick response. Maybe I should check some details further.
